### DISCUSSION TITLE
是不是强制新版主页了？

### DISCUSSION THREAD

**Discussion by Kal-tsit63** [the1812/Bilibili-Evolved#4450] [综合讨论 / General]
![image](https://github.com/the1812/Bilibili-Evolved/assets/85188780/64149831-a6cf-4806-b3fb-716618289723)

退回按钮没了，不过话说回来，这个样式以前是不是叫内测来着？
记忆中好像当初有三个版本分旧版，新版和内测来着，这个样式当初是内测

**Comment by Geeyun-JY3** [comment]
好像确实是没了按分区划分的那个可以拉到底的新版首页，现在默认都是这个无限滚动的内测首页了。
不过我这边暂时还能继续通过脚本维持在旧版首页，也不知道能撑多久。

**Reply by Kal-tsit63** [reply]
佬，插件能给一下吗，这个内测版看不习惯

**Reply by Geeyun-JY3** [reply]
https://github.com/the1812/Bilibili-Evolved/discussions/4206#discussioncomment-5938148

**Reply by Geeyun-JY3** [reply]
原来破站有这么个智能助手客服，今天偶然发现的，试了一下，刚好有这个问题的官方答案：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/112be77f-adf8-4a47-b573-3a6beedb5ad2)
